src/syntax_mapping.rs

```diff
@@ -1,3 +1,4 @@
+use assert_cmd::Command;
```

Comment:
It's probably easier to use an integration test instead of a unit test (in `tests/`). Otherwise, you need to move this import to the test code.

Reply:
Thanks, I have moved the import to the test code & now everything is working as expected & tests are passing.